examples/warp_specialize/example_warp_specialize_flashmla.py

```
@@ -17,12 +17,14 @@ def flashattn(batch, heads, kv_head_num, seqlen_kv, dim, pe_dim, block_N, block_
     assert kv_head_num == 1, "kv_head_num must be 1"
     h_dim = dim // 2
 
-    @T.macro
-    def flash_attn(
+    @T.prim_func
+    def main_no_split(
         Q: T.Tensor([batch, heads, dim], dtype),
         Q_pe: T.Tensor([batch, heads, pe_dim], dtype),
         KV: T.Tensor([batch, seqlen_kv, kv_head_num, dim], dtype),
         K_pe: T.Tensor([batch, seqlen_kv, kv_head_num, pe_dim], dtype),
+        glse: T.Tensor([batch, heads, num_split], dtype),
```

Ruff (0.14.10)

Unused function argument: `glse`

(ARG001)